Include/internal/pycore_code.h

```diff
@@ -609,12 +609,19 @@ PyAPI_DATA(const struct _PyCode8) _Py_InitCleanup;
 
 #ifdef Py_GIL_DISABLED
 
+static inline _PyCodeArray *
+_PyCode_GetTLBCArray(PyCodeObject *co)
+{
+    return _Py_STATIC_CAST(_PyCodeArray *,
```

Comment:
I thought about `_Py_STATIC_CAST` when I mentioned it in the issue, but the [precedent](https://github.com/search?q=repo%3Apython%2Fcpython%20_Py_STATIC_CAST&type=code) is to not use it for objects.